packages/shortest/src/types/browser.ts

```diff
@@ -13,23 +13,26 @@ export interface BrowserToolInterface {
   getPage(): Page;
 }
 
-export type BrowserAction =
-  | "mouse_move"
-  | "left_click"
-  | "left_click_drag"
-  | "right_click"
-  | "middle_click"
-  | "double_click"
-  | "screenshot"
-  | "cursor_position"
-  | "github_login"
-  | "clear_session"
-  | "type"
-  | "key"
-  | "run_callback"
-  | "navigate"
-  | "sleep"
-  | "check_email";
+export enum BrowserActionEnum {
+  MouseMove = "mouse_move",
+  LeftClick = "left_click",
+  LeftClickDrag = "left_click_drag",
+  RightClick = "right_click",
+  MiddleClick = "middle_click",
+  DoubleClick = "double_click",
+  Screenshot = "screenshot",
+  CursorPosition = "cursor_position",
+  GithubLogin = "github_login",
+  ClearSession = "clear_session",
+  Type = "type",
+  Key = "key",
+  RunCallback = "run_callback",
+  Navigate = "navigate",
+  Sleep = "sleep",
+  CheckMail = "check_mail",
+}
+
+export type BrowserAction = `${BrowserActionEnum}`;
```

Comment:
Why this change?

Reply:
I needed to use `screenshot` browser action and founded it no good to make hard code comparison. That's why I created enum from interface, not touching the interface itself. Now it possible to do like: `BrowserActionEnum.Screenshot`